static/app/components/replays/replayController.tsx

```diff
@@ -9,6 +9,8 @@ import {IconArrow, IconPause, IconPlay, IconRefresh, IconResize} from 'sentry/ic
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 
+import OptionSelector from '../charts/optionSelector';
```

Comment:
what's teh difference between optionSelector and `import CompactSelect from 'sentry/components/forms/compactSelect';`?

Reply:
Just the styles and some prop differences to get functional parity. I went ahead and refactored it